delocate/cmd/delocate_listdeps.py

```diff
@@ -38,10 +38,10 @@ def main():
         else:
             indent = ''
         if isdir(path):
-            lib_dict = tree_libs(path)
+            lib_dict = tree_libs_from_directory(path, ignore_missing=True)
             lib_dict = stripped_lib_dict(lib_dict, realpath(getcwd()) + psep)
         else:
-            lib_dict = wheel_libs(path)
+            lib_dict = wheel_libs(path, ignore_missing=True)
```

Comment:
Do we have a test for `ignore_missing` and `wheel_libs`?

Reply:
`ignore_missing` is passed directly to `tree_libs_from_directory`, and there are tests for it on that function.

If you're worried about gaps in testing then coverage testing could be setup to check for any.

Reply:
Right - but it is still possible for `wheel_libs` to fail to pass the parameter correctly.

Ouch, that is embarrassing that I didn't set up coverage.   I can do that.

Reply:
Since I can't test locally, coverage info would have to be displayed on an online site like `codecov.io`.  I can set it up myself if you have trouble with it.

Reply:
> Right - but it is still possible for wheel_libs to fail to pass the parameter correctly.

You could technically remove `ignore_missing=ignore_missing` from the `wheel_libs` function and there's no regression test for that yet.  I'll need to figure out an easier way to set up wheels with missing dependencies.

Reply:
I've finally added tests for `wheel_libs`'s `ignore_missing` parameter.